itests/kit/ensemble.go

```diff
@@ -321,6 +321,10 @@ func (n *Ensemble) MinerEnroll(minerNode *TestMiner, full *TestFullNode, opts ..
 		PresealSectors: presealSectors,
 		options:        options,
 		RemoteListener: rl,
+
+		cacheDirPath:       options.cacheDirPath,
+		unsealedSectorPath: options.unsealedSectorPath,
+		sealedSectorPath:   options.sealedSectorPath,
```

Comment:
if we're doing these in the ensemble then we should make it a map of `SectorNumber` to string so we can have more than one sector per miner